indico/modules/events/registration/client/styles/regform.module.scss

```diff
@@ -96,6 +96,17 @@
       list-style-type: none;
     }
   }
+
+  .checkbox-with-text {
+    input[type='checkbox'] {
+      float: left;
```

Comment:
I don't think we need floats here.. you can put the label next to the checkbox w/o the div wrapper and add a margin. In fact, AFAIK you can nest the checkbox *inside* the label and don't even need id/for in that case